src/components/ProjectDashboard/components/CyclesPayoutsPanel/components/ExportPayoutsCsvItem.tsx

```diff
@@ -19,29 +19,33 @@ export const ExportPayoutsCsvItem = ({
       ? fundingCycle.number.toNumber()
       : fundingCycle.number.toNumber() + 1
     : undefined
+  const disabled = !payoutSplits?.length
   const { exportSplitsToCsv } = useExportSplitsToCsv(
     payoutSplits ?? [],
     'payouts',
     fcNumber,
   )
 
-  if (!loading && !payoutSplits?.length) return null
-
   return (
-    <Button
-      type="text"
-      loading={loading}
-      className={twMerge(
-        'flex h-auto gap-2 p-4',
-        loading && 'cursor-not-allowed',
-      )}
-      style={{ color: 'inherit' }}
-      onClick={exportSplitsToCsv}
+    <Tooltip
+      title={t`No payouts to export`}
```

Comment:
don't we want this title to be conditional? seems like this will always show "no payouts"?

Reply:
This only shows when there is no payouts based on the `open={disabled...`